scripts/npm/create-component.js

```diff
@@ -162,7 +162,12 @@ const createComponent = async (err, { name }) => {
 
     fs.writeFileSync(storybookConfigFile, newStorybookConfigContent, 'utf8');
 
+    // Link everything up with Lerna.
+    console.log(colors.yellow(`Bootsrapping Lerna..`));
+    execSync(`npm run lerna bootstrap`);
+
     // Fix eslint errors and run Prettier.
+    console.log(colors.yellow(`Formatting code using eslint..`));
```

Comment:
Turns out `npm run create-component` was broken, so this fixes it. I tested that the fixes work by creating this component using it.